### DISCUSSION TITLE
Home Assistant su nuovo Mini PC --- Mille dubbi

### DISCUSSION THREAD

**Discussion by homeassistantby17** [indomus/forum#2808] [HUB: Home Assistant]
Buonasera, nella mia abitazione sto sperimentando ed utilizzando Home Assistant in esecuzione su una macchina virtuale VirtualBox installata su Windows 10.
Ora sto per acquistare un mini PC da dedicare **completamente ed unicamente** all'esecuzione di Home Assistant.
Nonostante abbia letto diverse guide sono molto combattuto sull'effettiva necessità di installare la versione Core su Docker al posto di quella OS, in primis per l'estrema facilità di gestione delle componenti aggiuntive, degli aggiornamenti e di tutto quanto rappresenta la versione OS.
Inoltre, mi sembra di intuire, che sull'installazione Core su Docker non potrei importare l'ultimo backup dell'installazione OS che attualmente sto utilizzando ma dovrei, invece, ripetere manualmente e da riga di comando tutte le installazioni e le configurazioni fatte sino ad oggi (ad esempio Mosquito, ZigbeeToMQQT, DuckDNS ecc. ecc.), giusto?
E' davvero così castrante affidarsi all'installazione della versione OS? Considerate che oggi, su macchina virtuale su macchina Windows 10 di qualche anno fa, con 2 GB di RAM dedicata a HA, non ho alcun tipo di problemi in merito a ritardi, lag o mancate automazioni.

Grazie mille in anticipo per i vostri commenti e suggerimenti.

**Comment by snakuzzo** [comment]
Va bene se resti su HA OS.
Poi se vuoi dedicare esclusivamente tutto il server oppure no, è una scelta tua (va bene anche lasciarlo su VM, non cambia assolutamente nulla).
Le osservazioni fatte su Core sono corrette...non hai gli addon e quindi dovresti preoccuparti di creare e integrare i servizi a parte.
Sul "castrante" dipende da quello che ti serve. Se dici che oggi stai bene su HA OS, continua ad usare HA OS e fine

**Reply by homeassistantby17** [reply]
Grazie mille Diego, stavo pensando di passare a un miniPC dedicato totalmente perché a volte il PC Windows viene riavviato per installare gli aggiornamenti ecc. ecc. e il servizio VM non sempre riparte in maniera corretta o, altre volte ancora, la VM crasha da sola e addio domotica at all.

**Reply by KC-inDomus** [reply]
ma infatti è senza senso usare macchine virtuali, proprio nonsense (in questo ambito, intendo)

**Comment by homeassistantby17** [comment]
> ma infatti è senza senso usare macchine virtuali, proprio nonsense (in questo ambito, intendo)

Per il momento uso questa soluzione solo perché ho già un PC desktop esistente always on che mi fa da server per altri servizi e l'unico modo per testare HA ed iniziare a fare un po' di domotica era la VM...il suo lavoro lo fa in maniera (per come serve a me) soddisfacente, a parte gli occasionali down causati da quanto ho descritto in precedenza e da qualche mancanza di risorse hardware.
